ms2pip/ms2pip_tools/spectrum_output.py

```diff
@@ -663,6 +664,68 @@ def write_bibliospec(self):
 
         return file_obj_ssl, file_obj_ms2
 
+    @output_format('dlib')
+    def write_dlib(self):
+        """
+        Write MS2PIP predictions to DLIB.
+        """
+        from ms2pip.ms2pip_tools.dlib import Entry, open_sqlite, metadata
+
+        normalization = "basepeak_10000"
+        precision = 5
+        if not self.normalization == normalization:
+            self._normalize_spectra(method=normalization)
+            self._generate_preds_dict()
+        if not self.peprec_dict:
+            self._generate_peprec_dict()
+        if precision not in self.diff_modification_mapping:
+            self._generate_diff_modification_mapping(precision)
+
+        filename = "{}.dlib".format(self.output_filename)
+        logger.info("writing results to %s", filename)
+
+        logger.warn("write mode is ignored for DLIB at the file mode, although append or not is respected")
```

Comment:
I guess this can just be a comment in the code, as it's more of an implementation detail, not really relevant to the user?

Reply:
Might be interesting still, maybe debug level as a compromise?

Reply:
Yup, perfect!